### PR TITLE
Add bundle destination for cmake install commands

### PR DESCRIPTION
**Pull request by anton-danielsson**
Fixes the problem:
"no BUNDLE DESTINATION for MACOSX_BUNDLE " when building for iOS

@ZhenLian

### CODE REVIEW

**Comment by anton-danielsson** [comment]
Let me know if there is anything more I need to do, or if there is some problem with the change?

File: CMakeLists.txt [hunk 1]
```diff
@@ -1088,6 +1088,7 @@ target_link_libraries(address_sorting
 if(gRPC_INSTALL)
   install(TARGETS address_sorting EXPORT gRPCTargets
     RUNTIME DESTINATION ${gRPC_INSTALL_BINDIR}
+    BUNDLE DESTINATION  ${gRPC_INSTALL_BINDIR}
```
Comment (jtattermusch) [inline CMakeLists.txt:1091]: the CMakeLists.txt is autogenerated from templates/CMakeLists.txt.template, so only updating this file won't work.

Comment (anton-danielsson) [reply CMakeLists.txt:1091]: Oh, I see :)

Comment (anton-danielsson) [reply CMakeLists.txt]: Should I update both or is the generated CMakeLists.txt updated by the CI/CD?

**Review by jtattermusch — APPROVED** [review]

**Pull request merged**